README.md

```diff
@@ -314,6 +314,49 @@ overrides the value specified in **Workspace**.
 | `independent` | `bool` | No | Yes | `version`, `publish` |
 | `no_individual_tags` | `bool` | Yes | No | `version`, `publish` |
 
+## Running tests
```

Comment:
Please move this to a CONTRIBUTING.md file and link it in contributors section in this README.